Jenkinsfile

```diff
@@ -421,10 +395,38 @@ stage('Test with sync') {
   )
 }
 
+def buildAndroidWrappers(abis, String stashName, Map extraCMakeArguments = [:]) {
```

Comment:
The `abis` we pass seem to be the same everywhere we use this, should we hardcode them instead?